MediaBrowser.Controller/Entities/PersonInfo.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using MediaBrowser.Model.Entities;
+
+namespace MediaBrowser.Controller.Entities
+{
+    /// <summary>
+    /// This is the small Person stub that is attached to BaseItems
```

Comment:
```suggestion
    /// This is a small Person stub that is attached to BaseItems.
```

Reply:
Why not "the" when it only one?

Reply:
There's only one per person but many in the database right?

Reply:
🤷‍♂